src/operators/subgraph.c

```diff
@@ -359,6 +359,57 @@ igraph_error_t igraph_i_induced_subgraph_map(const igraph_t *graph, igraph_t *re
     }
 }
 
+/**
+ * \ingroup structural
+ * \function igraph_induced_subgraph_map
+ * \brief Creates an induced subraph and returns the mapping from the original.
+ *
+ * </para><para>
+ * This function collects the specified vertices and all edges between
+ * them to a new graph.
+ * As the vertex IDs in a graph always start with zero, this function
+ * very likely needs to reassign IDs to the vertices.
+ * \param graph The graph object.
+ * \param res The subgraph, another graph object will be stored here,
```

Comment:
Suggestion: rename this param to `subgraph`, both here and in the function signature.